zhaquirks/sinope/sensor.py

```diff
@@ -30,7 +30,8 @@
     OUTPUT_CLUSTERS,
     PROFILE_ID,
 )
-from zhaquirks.sinope import SINOPE, SINOPE_MANUFACTURER_CLUSTER_ID
+
+from . import SINOPE, SINOPE_MANUFACTURER_CLUSTER_ID
```

Comment:
Please revert to using absolute imports everywhere. It's easier to use them as custom quirks then.
Background: https://github.com/zigpy/zha-device-handlers/pull/827

Reply:
It's quite the opposite. Without them I can't use this as a quirk. Since there are modifications to the `__init__`. Without them you'll be using the wrong init file. I definitely challenge the call to use absolute imports for current module. I don't understand what's easier either. My setup to include this as a quirks is very simple.

Reply:
Ah, so you're importing all these via the "custom quirks" feature.
If you do bigger modifications (involving multiple classes/files), it's ideal to directly modify the `zhaquirks` package.

We need absolute imports, as we want users to be able to easily modify a single quirk file (that imports from other files), without needing to load everything as custom quirks. For example, to easily add a single model to `MODELS_INFO`.
So, these changes need to be reverted before we can merge it.

Reply:
done